integration_tests/test_dbapi.py

```diff
@@ -47,7 +47,6 @@ def test_select_query(presto_connection):
     rows = cur.fetchall()
     assert len(rows) > 0
     row = rows[0]
-    assert row[0] == "test"
```

Comment:
the test query above should be using concrete column names instead of `*` (but that would be a separate commit)

i don't understand what `row[0]` was referring to and why it gets removed

Reply:
It was node_id specified in etc/node.properties. It's not a fixed value after this commit. 
```
presto> select * from system.runtime.nodes;
   node_id    |        http_uri        | node_version | coordinator | state
--------------+------------------------+--------------+-------------+--------
 b8c38cccce9d | http://172.17.0.2:8080 | 329          | true        | active
```